data/logpoint-nepal.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "LogPoint Nepal",
+  "establishedAt": 1041358500,
+  "location": "Jawalakhel, Lalitpur, Nepal",
+  "website": "https://www.logpoint.com/",
+  "email": null,
+  "phone": "+977-15541326",
+  "socials": {
+    "facebook": "https://www.facebook.com/LogpointSIEM",
+    "github": null,
+    "instagram": "https://www.instagram.com/explore/tags/logpointnepal/",
```

Comment:
Let's not have `Instagram` if they don't have a dedicated instagram page.

Reply:
Removed the link in that case.